tests/test-cases/Group1-Docker-Commands/TestCases.md

```diff
@@ -47,4 +47,6 @@ Group 1 - Docker Commands
 [Test 1-22 - Docker Volume RM](1-22-Docker-Volume-RM.md)
 -
 [Test 1-23 - Docker Inspect](1-23-Docker-Inspect.md)
--
\ No newline at end of file
+-
+[Test 1-24 - Docker Link](1-24-Docker-Link.md)
```

Comment:
I think they generated automatically but I might be wrong, @mhagen-vmware help us :)